lib/StaticNarrative/narrative/updater.py

```diff
@@ -1,499 +0,0 @@
-"""
```

Comment:
nothing from this file gets used ==> bye bye!

Reply:
Yeah, I remember putting that in there to try to auto-update from a possibly older Narrative object version that a user might try to make a Static Narrative from. Then I guess I just figured on 1. avoiding code creep from copy-pasta, and 2. letting the Static process just crash and tell the user they're using a very old Narrative and they should make a new save first. And never got around to deleting this.